src/index.ts

```diff
@@ -28,3 +28,15 @@ export function customScanner(scanOptions: ScanOptions) {
     localScannerCli: true,
   });
 }
+
+function scanWithCallback(scanOptions: ScanOptions, callback: (error?: any) => void) {
+  return scan(scanOptions)
+    .then(() => {
+      callback();
+    })
+    .catch(error => {
+      callback(error);
+    });
```

Comment:
Added a `catch` otherwise the callback would never get called when an error occurs